File: aifupan_project/ReviewAnalysis/vo/AnalysisResultCruxTypeVo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ReviewAnalysis.vo
{
    public class AnalysisResultCruxTypeVo
    {
        /// <summary>
        /// 关键词分类信息
        /// </summary>
        public CruxTypeInfoVo cruxTypeInfoVo { get; set; }

        /// <summary>
        /// 分类关键词占比比例值，0.21表示21%
        /// </summary>
        public double scale { get; set; }

        /// <summary>
        /// 分类关键词数量
        /// </summary>
        public int num { get; set; }

    }
}

File: aifupan_project/fupan-server/replay-generic/src/main/java/com/jiuyu/replay/generic/vo/words/AnalysisResultCruxTypeVo.java
package com.jiuyu.replay.generic.vo.words;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

/**
 * 分析结果的关键词类型信息
 */
@Data
@Schema(description = "分析结果的关键词类型信息")
public class AnalysisResultCruxTypeVo {

    /**
     * 关键词分类信息
     */
    @Schema(description = "关键词分类信息")
    private CruxTypeInfoVo cruxTypeInfoVo;
    /**
     * 分类关键词占比比例值，0.21表示21%
     */
    @Schema(description = "占比比例值，0.21表示21%")
    private Double scale;
    /**
     * 分类关键词数量
     */
    @Schema(description = "分类关键词数量")
    private Integer num;
}
